src/Tizen.NUI/src/public/Window/WindowEvent.cs

```diff
@@ -85,6 +86,8 @@ public partial class Window
         private delegate void MoveCompletedEventCallbackType(IntPtr window, IntPtr position);
         [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
         private delegate void ResizeCompletedEventCallbackType(IntPtr window, IntPtr size);
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
```

Comment:
thank you for your fix.
but could you change this as `.Cdecl` ?

Reply:
Thank you for your comment!
I updated the code as you requested.

Reply:
thank you for fix